app/src/main/java/org/mozilla/fenix/sync/SyncedTabsFragment.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.sync
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.fragment_synced_tabs.*
+import mozilla.components.browser.storage.sync.Tab
+import mozilla.components.feature.syncedtabs.SyncedTabsFeature
+import mozilla.components.support.base.feature.ViewBoundFeatureWrapper
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.library.LibraryPageFragment
+
+class SyncedTabsFragment : LibraryPageFragment<Tab>() {
+    private val syncedTabsFeature = ViewBoundFeatureWrapper<SyncedTabsFeature>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        return inflater.inflate(R.layout.fragment_synced_tabs, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val backgroundServices = requireContext().components.backgroundServices
+
+        syncedTabsFeature.set(
+
+            feature = SyncedTabsFeature(
+                storage = backgroundServices.syncedTabsStorage,
+                accountManager = backgroundServices.accountManager,
+                view = synced_tabs_layout,
+                lifecycleOwner = this,
```

Comment:
The Fragment lifecycle doesn't necessarily match with the Fragment view lifecycle, so here we should probably be using `this.viewLifecycleOwner` to make sure this feature lives for onViewCreated -> onViewDestroyed